src/sentry/tasks/unmerge.py

```diff
@@ -54,7 +57,7 @@ def fetch(*key):
     return fetch
 
 
-def get_caches():
+def get_caches() -> Caches:
```

Comment:
this should probably get refactored or deleted instead -- this isn't typesafe as written

I think most (all?) of this can be replaced entirely with `@functools.lru_cache` -- or really just avoided entirely and use django's builtin caches

Reply:
possibly - let's get some types in this file before we refactor though 😅

Reply:
these types won't really help you -- all calls are untyped here